src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -193,6 +213,16 @@ class TinyMCEConfig extends HTMLEditorConfig implements i18nEntityProvider
      */
     private static $base_dir = null;
 
+    /**
+     * Location of tinymce translation files relative to BASE_DIR.
+     *
+     * Supports vendor/module:path
+     *
+     * @config
+     * @var string
+     */
+    private static $lang_dir = null;
```

Comment:
Needed because they no longer come packages with tinymce, so we need to include them separately and point to wherever they are